lib/account/pages/about_page.dart

```diff
@@ -9,15 +12,101 @@ class AboutPage extends StatefulWidget {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageAboutTitle),
       ),
       body: Center(
-        child: SingleChildScrollView(
-          child: Text(S.of(context).appName),
+        child: CustomScrollView(
+          physics: const ClampingScrollPhysics(),
+          slivers: [
+            SliverFillRemaining(
+              hasScrollBody: false,
+              child: Padding(
+                padding: const EdgeInsets.all(10),
+                child: Column(children: [
+                  Expanded(child: Container()),
+                  Text(
+                    S.of(context).appName,
+                    style: Theme.of(context).primaryTextTheme.displaySmall,
+                  ),
+                  if (_packageInfo != null) ...[
+                    Text(
+                      S.of(context).pageAboutVersion(_packageInfo!.version),
+                      style: Theme.of(context)
+                          .primaryTextTheme
+                          .subtitle2!
+                          .copyWith(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
+                      textAlign: TextAlign.center,
+                    ),
+                    Text(
+                      S.of(context).pageAboutBuildNumber(_packageInfo!.buildNumber),
+                      style: Theme.of(context)
+                          .primaryTextTheme
+                          .subtitle2!
+                          .copyWith(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
+                      textAlign: TextAlign.center,
+                    ),
+                    Text(
+                      S.of(context).pageAboutBuildSignature(_packageInfo!.buildSignature),
+                      style: Theme.of(context)
+                          .primaryTextTheme
+                          .subtitle2!
+                          .copyWith(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
+                      textAlign: TextAlign.center,
+                    ),
+                  ],
+                  SvgPicture.asset(
+                    'assets/logo.svg',
+                    width: MediaQuery.of(context).size.width / 2,
+                    height: MediaQuery.of(context).size.width / 2,
+                    semanticsLabel: 'MOTIS Logo',
+                  ),
+                  Text(
+                    S.of(context).pageAboutText,
+                    textAlign: TextAlign.center,
+                  ),
+                  Expanded(
+                    child: Align(
+                      alignment: Alignment.bottomCenter,
+                      child: Row(mainAxisAlignment: MainAxisAlignment.center, children: [
+                        TextButton(
+                          onPressed: () async => await launchUrlString(
+                            'https://github.com/MOTIS-Mitfahr-App/flutter-app',
+                            mode: LaunchMode.externalApplication,
```

Comment:
why does this have another launch mode, but not the motis links?

Reply:
The default (at least on Android) is that the link opens a kind of weaker in-app browser without tabs or anything that also doesn't cache where it was when the app is closed. I really like that for the website since you don't get the feeling you are leaving the app, but I thought if you want to browse the GitHub project you will want to have it open in your normal browser, especially so that you can keep browsing after the app is closed